02/app.js

```diff
@@ -1,2 +1,15 @@
 const x = 5;
-let result = 0;
\ No newline at end of file
+let result = 0;
+
+for (let i = 0; i <= x; i++) {
+    result = result + i
+}
+console.log(result)
+
+// while loop
+let result2 = 0;
+
+while(result2 !== 15 ) {
```

Comment:
Tutaj należało wstawić również `x` zamiast 15 - jest to bardziej elastyczne rozwiązanie. Zmieniamy `x` i obie pętle działają tak samo.